subscription/src/test/java/org/killbill/billing/subscription/alignment/TestPlanAligner.java

```diff
@@ -56,6 +59,12 @@ public void beforeClass() throws Exception {
         planAligner = new PlanAligner();
     }
 
+    @BeforeMethod(groups = "fast")
+    public void beforeMethod() throws Exception {
+        super.beforeMethod();
```

Comment:
You also need the `if (hasFailed()) {return;}` dance.

Reply:
ack